src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/ApplicationsTransformerTest.kt

```diff
@@ -179,6 +183,98 @@ class ApplicationsTransformerTest {
     assertThat(result.status).isEqualTo(ApplicationStatus.submitted)
   }
 
+  @Test
+  fun `transformJpaToApi transforms an Approved Premises application with a rejected Assessment correctly`() {
```

Comment:
I feel like the names of these tests would be better with something along with lines of the following rather than just saying "transforms correctly":

```
transformJpaToApi sets an Approved Premises application's status to "rejected" if the associated assessment has a decision of "rejected"
```